src/mo_frontend/typing.ml

```diff
@@ -3427,6 +3469,16 @@ and infer_block_exps env decs : T.typ =
 and infer_dec env dec : T.typ =
   let t =
   match dec.it with
+  | IncludeD (i, arg, n) ->
+    if not env.pre then begin
+      use_identifier env i.it;
+      if not env.in_actor then
```

Comment:
Note to self: This is the place to reject includeD with mixins, if we want - maybe by just changing in_actor to in_obj_sort... (or adding in_mixin)